src/service.c

```diff
@@ -76,6 +76,8 @@ static const char * const menu_names[N_PROFILES] =
   "desktop_lockscreen"
 };
 
+static const char * const usage_mode_schema_name = "com.canonical.Unity8";
```

Comment:
As of the latest Lomiri rename MR [1][2], the schema is now renamed to `com.lomiri.Shell`. However, the rename is not exactly finalized yet. I don't expect this one to change, but just in case.

[1] https://gitlab.com/ubports/core/lomiri/-/merge_requests/2
[2] https://gitlab.com/ubports/core/lomiri/-/merge_requests/8

Reply:
Please also adjust the commit message (although we change the original metadata with this). It is Lomiri, not Unity8...